great_expectations/checkpoint/actions.py

```diff
@@ -158,7 +165,9 @@ def __init__(
                 class_name=renderer["class_name"],
             )
         self.slack_webhook = slack_webhook
-        assert slack_webhook, "No Slack webhook found in action config."
+        self.slack_token = slack_token
+        self.slack_channel = slack_channel
+        # assert slack_webhook, "No Slack webhook found in action config."
```

Comment:
We should still be able to `assert` this if `slack_token` and `slack_channel` are `None`

Reply:
And vise-versa if `slack_webhook` is `None` we can `assert` that the other two are provided